Bottom: df8e8e01bbab8c53a11220806d3b2597a5c75e97
Top:    4a4ef7bc44a35c32d7baddbed961d401730fe7ac
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:53:08 +0800

Refresh of autofs-5.0.5-add-master-read-wait-option.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index ff7b0ad..f5aa287 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -49,6 +49,7 @@
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
+- wait for master map available at start.
 - add master read wait option.
 
 25/07/2012 autofs-5.0.7
